extensions/Khronos/KHR_draco_mesh_compression/README.md

```diff
@@ -0,0 +1,141 @@
+# KHR_draco_mesh_compression
+
+## Contributors
+
+* Fan Zhang, Google, <mailto:zhafang@google.com>
+* Ondrej Stava, Google, <mailto:ostava@google.com>
+* Frank Galligan, Google, <mailto:fgalligan@google.com>
+* Kai Ninomiya, Google, <mailto:kainino@google.com>
+* Patrick Cozzi, Cesium, [@pjcozzi](https://twitter.com/pjcozzi)
+
+## Status
+
+Draft
+
+## Dependencies
+
+Written against the glTF 2.0 spec.
+
+## Overview
+
+This extension defines a schema to use [Draco geometry compression](https://github.com/google/draco) libraries in glTF format. This allows glTF to support streaming compressed geometry data instead of the raw data.
+
+The [conformance](#conformance) section specifies what an implementation must do when encountering this extension, and how the extension interacts with the attributes defined in the base specification.
+
+## glTF Schema Updates
+
+Draco geometry compression library could be used for `primitive` by adding an `extension` property to a primitive, and defining its `KHR_draco_mesh_compression` property.
+
+The following picture shows the structure of the schema update. 
+
+**Figure 1**: Structure of geometry compression extension.
+![](figures/structure.png)
+
+In general, we will use the extension to point to the buffer that contains the compressed data. The major change is that the `accessor` in an extended `primitive` no
+longer points to a `bufferView`. Instead, the `attributes` of a primitive will use the decompressed data. This is valid because in glTF 2.0, `bufferView` is not required in `accessor`, although if it is not present, it will be used with `sparse` field to act as a sparse accessor. In this extension, we will ignore the `bufferView` property.
+
+Usage of the extension must be listed in the `extensionUsed` and `extensionsRequired`. 
```

Comment:
Sounds good on the first point.

I don't think I understand the second. Taking this example again, with four uncompressed attributes:

```
"attributes": {
    "POSITION" : 11,
    "NORMAL" : 12,
    "TEXCOORD_0" : 13,
    "TEXCOORD_1" : 14
}
```

And let's assume the Draco extension defines compressed versions of the first three, but not `TEXCOORD_1`. Then, the exporter should put attributes 1-3 into one buffer, `TEXCOORD_1` into another, and the Draco data into a third buffer. Loaders supporting Draco will request buffers 2+3, and loaders not supporting Draco will load buffers 1+2.

But, that choice of buffers should probably be an implementation note and not part of the spec. It would be technically valid for an exporter to put all the compressed and uncompressed data into one buffer, even though that is inefficient.